packages/core-components/src/layout/SignInPage/SignInPage.tsx

```diff
@@ -151,9 +149,12 @@ export const SingleSignInPage = ({
       setShowLoginPage(true);
     }
   };
-  if (loginCount === 0) {
+
+  useEffect(() => {
```

Comment:
Think you could actually use the `useMount` hook here instead of adding the exhaustive deps eslint rule exception.

`import { useMount } from 'react-use'` I think. And it should already be a dependency.

Reply:
oh nice! Done